proto/asgt/dataservice/v1alpha1/bq_entries.proto

```diff
@@ -24,7 +24,7 @@ message BankEntry {
   repeated string tags = 4 [(gen_bq_schema.bigquery).description = "Tags defined by consumer, enriched by service if valetkey is used"];
   asgt.type.BankTransaction data = 5;
   repeated asgt.type.TargetValue target_values = 6;
-  asgt.type.ModelInfo model = 9;
+  asgt.type.Model model = 9;
```

Comment:
Do we store the model info in BQ?

Reply:
Yes, currently we store the version number. This would add the created_at date too.

Reply:
:+1: